charts/redpanda/values.yaml

```diff
@@ -252,6 +271,120 @@ storage:
     labels: {}
     # Additional annotations to apply to the created PersistentVolumeClaims.
     annotations: {}
+  # Tiered Storage helps to lower storage costs by offloading log segments to cloud storage.
+  # You can specify the amount of local storage you want to retain in local storage.
+  # You don't need to specify which log segments you want to move because Redpanda moves them automatically
+  # based on cluster-level configuration properties. Redpanda Tiered Storage indexes where data is offloaded,
+  # so it can retrieve the data when you need it.
+  #
+  # REF: https://docs.redpanda.com/docs/manage/tiered-storage/#tiered-storage-configuration-properties
+  #
+  # Topic configuration flags override any cluster-level configuration. For example, on new topics,
+  # if `cloud_storage_enable_remote_write` is set to `true`, you can set `redpanda.remote.write` to `false`
+  # to turn it off for a particular topic. Or, if you have Tiered Storage turned off for the cluster, you can enable it
+  # for individual topics with the topic flags. When cluster-level properties are changed,
+  # the changes apply only to new topics, not existing topics.
+  #
+  # REF: https://docs.redpanda.com/docs/manage/tiered-storage/#enable-tiered-storage-for-a-topic
+  tieredConfig:
+    # Global flag that enables Tiered Storage feature if license is provided.
+    # Default is false.
+    cloud_storage_enabled: false
+    # Cluster wide config option to enable data upload from Redpanda to cloud storage if set to true.
+    cloud_storage_enable_remote_write: true
+    # Cluster wide config option to enable Redpanda to fetch data from cloud storage if set to true.
+    cloud_storage_enable_remote_read: true
+    # This configuration option is available starting from 22.3.X
```

Comment:
If the comment is referring the the previous configuration, let's move it above the config so that it is clear.

Reply:
This lines refers to https://github.com/redpanda-data/helm-charts/blob/1e53583ba306ff64d073a86e05e1ce0321b49a91/charts/redpanda/values.yaml#L306

I will change the comment to replace word `this` with `cloud_storage_credentials_source`